dcpy/connectors/edm/publishing.py

```diff
@@ -14,38 +14,42 @@
 ## TODO - these functions need some cleaning
 ## ideally they don't need to called individually, but we either have consistent folder structure
 ## Or have metadata in json/etc about structure of outputs per dataset
-def get_dataset_branch_path(dataset: str, branch: str, version: str):
-    return f"{BASE_URL}/{dataset}/{branch}/{version}/output"
-
-
-def get_dataset_path(dataset: str, version: str):
-    return f"{BASE_URL}/{dataset}/{version}/output"
+def get_dataset_path(dataset: str, *, branch: str, version="latest", published=False):
```

Comment:
nit: should the `*` be after the last param without a default value? so
```python
def get_dataset_path(dataset: str, branch: str, *, version="latest", published=False):
````

Reply:
That would have the effect of making it a required positional variable, which we probably don't want to do. All the product pathing is in flux atm, so maybe we revisit once all releases are in the `publish` folder